src/compat/cxx/core/cxx_app_startup_shutdown.cpp

```diff
@@ -14,7 +14,6 @@ int cxx_app_start(int argc, char **argv) {
 	int ret;
 
 	cxx_app_startup();
-	getopt_init();
 	ret = main(argc,argv);
 	cxx_app_termination();
```

Comment:
This one requires additional investigation, let's do not remove in this PR. I'd propose to add an issue for these exceptional cases and solve it then.

Reply:
ok, everything's done, I think. I've updated the PR, excluding those files which require further digging. Also created a new Issue for them here #1863 .

Reply:
Yes, thanks. Only a question about `getopt_init` has remained.